gitsome/config.py

```diff
@@ -643,6 +652,15 @@ def save_config(self):
             parser.set(self.CONFIG_SECTION,
                        self.CONFIG_VERIFY_SSL,
                        self.verify_ssl)
+            # Update existing config files without disable_avatar correctly
+            if self.disable_avatar is not None:
+                parser.set(self.CONFIG_SECTION,
+                           self.CONFIG_DISABLE_AVATAR,
+                           self.disable_avatar)
+            else:
+                parser.set(self.CONFIG_SECTION,
```

Comment:
In what cases would this else loop be executed?  We're setting a default in `__init__` and calling `load_config` with a default value.